cli/azd/pkg/output/ux/uxitem.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package ux
+
+import "github.com/azure/azure-dev/cli/azd/pkg/output"
+
+type UXItem interface {
+	// Defines how the object is transformed into a printable string.
+	// The current indentation can be used to make the string to be aligned to the previous lines.
+	ToString(currentIndentation string) string
```

Comment:
It is my belief that the current indentation should be the state on the console..